components/MashCards/InitialDecisionCard.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import { mockedMashReferral } from 'factories/mashReferral';
+import InitialDecisionCard from './InitialDecisionCard';
+
+describe('InitialDecisionCard', () => {
+  it('renders the right info from the mash referral', () => {
+    render(<InitialDecisionCard mashReferral={mockedMashReferral} />);
+
+    expect(screen.getByText('4 hours left'));
+    expect(screen.getByText(`${mockedMashReferral.clients[0]} (referral)`));
+    expect(screen.getByText(mockedMashReferral.screeningDecision as string));
```

Comment:
an initial decision (and by extension initial decision card) does not have a screening decision.

It should have a requested support that we want to check for.

Reply:
Good point - didn't see this!